Rake docker:build task uses docker compose subcommand (#15462)
openSUSE/open-build-service: CONTRIBUTING.md

that require an case sensitive filesystem (default in Linux, configurable in **MacOS/Windows**),
make sure you run all this from a filesystem that supports this. Here you have [some instructions](https://github.com/openSUSE/open-build-service/wiki/Setup-an-OBS-Development-Environment-on-macOS) in case you are a MacOS user.

1. Install [docker](https://www.docker.com)
   There is documentation about this for [openSUSE](https://en.opensuse.org/SDB:Docker) and various
   [other operating systems](https://docs.docker.com/engine/installation/).

>    ¹ A greater than _2.00_ is required for _docker compose_ subcommand 

Reviewer: ```suggestion
   ¹ A version greater than _2.00_ is required for _docker compose_ subcommand 
```

Author: Fixed

that require an case sensitive filesystem (default in Linux, configurable in **MacOS/Windows**),
make sure you run all this from a filesystem that supports this. Here you have [some instructions](https://github.com/openSUSE/open-build-service/wiki/Setup-an-OBS-Development-Environment-on-macOS) in case you are a MacOS user.

> 1. Install [docker](https://www.docker.com)

Reviewer: ```suggestion
1. Install [docker (version >= v2)¹](https://www.docker.com)
```
We should not forget the reference described with the `¹`.

And also, shouldn't be the minimum required version  `v2`?

Author: Yes, it should be

   There is documentation about this for [openSUSE](https://en.opensuse.org/SDB:Docker) and various
   [other operating systems](https://docs.docker.com/engine/installation/).

>    ¹ A version greater than _2.00_ is required for _docker compose_ subcommand to work.

Reviewer: ```suggestion
   ¹ A version greater than _v2_ is required for _docker compose_ subcommand to work.
```
Only for consistency...
